ansible.cfg

```diff
@@ -2,7 +2,7 @@
 callback_plugins = $PWD/playbooks/callback_plugins/
 nocows = 1
 inventory = $PWD/playbooks/inventory/vagrant.py
-roles_path = $PWD/playbooks/roles
+roles_path = $PWD/galaxy_roles:$PWD/playbooks/roles
```

Comment:
I was thinking of putting this side-by-side for parity: `playbooks/galaxy_roles`